### PR TITLE
Add workflow to automatically generate pull requests for the latest MapLibre GL JS version

### PR DESCRIPTION
**Pull request by wipfli**
This pull request adds a GitHub Action which 

* checks out the docs repos, 
* updates the submodule in `maplibre-gl-js` to the latest commit,
* creates a branch named with the new version (e.g. `v1.15.2`), 
* pushes the branch to `maplibre/maplibre-gl-js-docs`, 
* and creates a pull request to merge the new branch into `main`.

### CODE REVIEW

File: .github/workflows/update-gl-js.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    - run: git checkout -b v$(npm show maplibre-gl version)
+    - run: |
+        git submodule update --init
+        cd maplibre-gl-js
+        git checkout main
+        git pull
+        cd ..
+        git add -u
+        git commit -m "Update MapLibre GL JS to v$(npm show maplibre-gl version)"
+    - run: git push --set-upstream origin v$(npm show maplibre-gl version)
```
Comment (nyurik) [inline .github/workflows/update-gl-js.yml:26]: does this step require eny github tokens?

**Comment by wipfli** [comment]
https://github.com/maplibre/maplibre-gl-js/pull/200

**Comment by wipfli** [comment]
https://github.com/maplibre/maplibre-gl-js-docs/pull/63

File: .github/workflows/update-gl-js.yml (hunk 1 above)
Comment (wipfli) [reply .github/workflows/update-gl-js.yml:28]: No, this one does not.

**Review by nyurik — APPROVED** [review]
lgtm!

**Comment by wipfli** [comment]
Thanks for reviewing Yuri and for the constructive feedback.

**Pull request merged**